### ISSUE TITLE
dotty

### ISSUE THREAD

**Issue by arnolddevos** [scalasyd/scalasyd#43]
I find this development of scala interesting, which I am following via Odersky's various talks and his paper. (Most recently: ScalaDays NYC. http://www.slideshare.net/Odersky/scala-days-nyc-2016 )  Perhaps others are also interested?  A talk from someone who has played with dotty would be great.

**Comment by karlroberts** [comment]
+1

**Comment by kevin-lee** [comment]
@arnolddevos Do you think you can give a talk on Dotty next week? As you know, it was announced that [Dotty will become Scala 3.0](https://www.scala-lang.org/blog/2018/04/19/scala-3.html) so I think it's a good time to talk about Dotty.

**Comment by arnolddevos** [comment]
@Kevin-Lee Hi Kevin, I am overseas next week otherwise I would be willing.

**Comment by arnolddevos** [comment]
I am thinking of a clickbait title for such a talk e.g. Dotty - Functional Programming with Implicit Mutable Data.  I could go on.    

Actually "How to Use a Monad for Functional Programming with Mutable State: Haskell vs. scala vs. dotty" is a perfectly reasonable topic.

**Comment by kevin-lee** [comment]
@arnolddevos That sounds good and I'm sorry I was wrong about the date. It's this week and too late to organise it. So do you think you can do it next month?

**Comment by arnolddevos** [comment]
I can't do it for July either.  Sorry about that.   (I did not realise I would be unavailable
 when I suggested titles in the previous comment.)

**Comment by kevin-lee** [comment]
@arnolddevos That's fine. Could you do it next month then (The 8th of August)?

**Comment by arnolddevos** [comment]
@Kevin-Lee OK.  Will put my best foot forward next month.

**Comment by kevin-lee** [comment]
@arnolddevos Hi Arnold, the upcoming ScalaSyd will be on the 8th of August.
Could you please confirm that you can give a talk on dotty?
Thanks!

**Comment by arnolddevos** [comment]
Hi Kevin, yes I will.   I had a look at it yesterday and I think it would be interesting to cover a couple of aspects of dotty with examples.

**Comment by arnolddevos** [comment]
If you are announcing the meetup I would now suggest a more broad title for this talk.   Something like "Interesting language enhancements in dotty/scala 3".

**Comment by arnolddevos** [comment]
Is the meetup going to take place on August 8?   I just checked meetup.com and could not find it.

**Comment by jedws** [comment]
yes, I'm just in the process of putting together the meetup announcement

**Comment by jedws** [comment]
@arnolddevos I don't seem to have a description for this, can you add one and I'll add to meetup description? Thanks!

**Comment by arnolddevos** [comment]
@jedws OK I see the announcement now.  Please excuse my gentle prodding.

**Comment by arnolddevos** [comment]
@jedws re description:

A tour of some selected new language features that can be seen in the latest dotty RC (and will probably be in scala 3).  To make it interesting we will try to use these features in example code. Hopefully some of their raison d'être will emerge.

**Comment by kevin-lee** [comment]
Came back from overseas today. Thanks @arnolddevos. I'm looking forward to hearing your talk this Wednesday.

**Comment by arnolddevos** [comment]
@Kevin-Lee Thanks.  I see there are three talks now.  Given the time and effort required, that's hard  on the third speaker. The scalasyd audience don't generally hang around that late.

**Comment by kevin-lee** [comment]
@arnolddevos Yeah I think so too. I think two might be ideal. Even myself can't stay for the third one as it takes me about one and half hours to get home.

**Comment by jedws** [comment]
my talk is only short (25 mins), so we should be able to get it done. I can
delay till another night if preferred though.

**Comment by arnolddevos** [comment]
I completely happy to postpone my talk.  It will still be relevant next month.   Initially, there were no talks and we pulled this one out of the suggestions archive.  Now that I have developed the talk I would like to give it eventually of course.

**Comment by jedws** [comment]
nah, my talk isn't even really Scala. You're still up :-)

**Comment by arnolddevos** [comment]
BTW , loved your delivery of Juliet's speech.  You were in the moment then - spine tingling.  More art in tech conferences I say.

**Comment by jedws** [comment]
awww, shucks!
